hermes.py

```diff
@@ -35,7 +35,7 @@ class Protocol(Enum):
 @click.option('--interval', '-i', type=int, help='frequency to poll all configured files, in seconds', default=40)
 @click.option('--jitter', '-j', type=int, help='each sleep will randomly add between 0 and j seconds', default=40)
 @click.option('--yamlfile', '-y', type=click.Path())
-@click.option('--secret-key-file', '-k', type=str, help='secret key for decrypting downloaded yaml file', multiple=True)
+@click.option('--secret-key-file', '-k', type=list, help='secret key for decrypting downloaded yaml file', multiple=True)
```

Comment:
I think the type here still needs to be a str, and you will need to do the comma split yourself. 

ATM it is splitting it into an array of characters when set this way.